collector/nvml/nvidia-smi.h

```diff
@@ -0,0 +1,26 @@
+#ifndef NVIDIA_SMI_H
+#define NVIDIA_SMI_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct {
```

Comment:
Isn't this struct defined in the nvml sdk?

Reply:
No, it is created for reading the data we need.

Reply:
No, it is created for reading the data we need.